bookshelf/routes/web.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/*
+ * Copyright 2019 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+$projectId = getenv('GOOGLE_CLOUD_PROJECT');
+$collectionName = getenv('FIRESTORE_COLLECTION') ?: 'books';
+
+# [START firestore_client]
+// Use the client library to call Firestore
+use Google\Cloud\Firestore\FirestoreClient;
+$firestore = new FirestoreClient([
+    'projectId' => $projectId,
+]);
+$collection =  $firestore->collection($collectionName);
+# [END firestore_client]
+
+# [START cloud_storage_client]
+// Use the client library to call Cloud Storage
+use Google\Cloud\Storage\StorageClient;
+$storage = new StorageClient([
+    'projectId' => $projectId,
+]);
+$bucketId = $projectId . '.appspot.com';
+$gcsBucket = $storage->bucket($bucketId);
+# [END cloud_storage_client]
+
+/*
+|--------------------------------------------------------------------------
+| Application Routes
```

Comment:
A number of the application route snippets do not have much context on the variables they use. Also, more region tag namespacing.